abaverify/processresults.py

```diff
@@ -45,6 +45,9 @@
     Finds the slope of an x-y data curve.
 7. finalValue
     Last output value.
+8. tabular
+    Given a list of (x,y) points confirms that a point, within the specified, tolerance exists in post processes
+    results.
```

Comment:
Change to:
Compares the values for a list of tuples specifying x, y points [(x1,y1), (x2, y2), ...]

Reply:
Updated